R/frontline_dm.R

```diff
@@ -33,7 +46,7 @@ col.date <- function(df){
 #' @return data.frame
 col.mod_date <- function(df, 
                          start_date = as.Date("07082021", "%m%d%Y"),
-                         max_date = as.Date("08042021", "%m%d%Y")){
+                         max_date = as.Date("11162021", "%m%d%Y")){
```

Comment:
I'm assuming this is some weird thing we have to keep updating because of Bowties's system?

Reply:
@holmesjoli Yes, although in this version, max_date is the same as export_date (from parse_date.frontline), so that might be an opportunity to simplify things. Not sure if that will be the case in the future though.